tygo/config.go

```diff
@@ -91,8 +103,8 @@ func (c Config) PackageConfig(packagePath string) *PackageConfig {
 
 func normalizeFlavor(flavor string) (string, error) {
 	switch flavor {
-	case "", "default":
-		return "default", nil
+	case "", "default", "undefined":
```

Comment:
I think this change is accidental and should be reverted (this logic is now the same as `normalizeOptional` below)

Reply:
Yes, I first had it the same as your proposal, but I think that defining optional as "undefined" in the config should be supported.
Whats your take on that?

Reply:
I agree it's the right choice - and there is identical code in the other function. I just think this `normalizeFlavor` function is the wrong one to edit as it's unrelated to this change

Reply:
My bad, I messed that up and sadly misunderstood your comment 😄 Should be fixed now

Reply:
No worries :)